lib/flight.js

```diff
@@ -48,6 +48,15 @@ const fetchFlight = async (flight) => {
 	let delay = time.historical?.delay ?? null
 	if (delay) delay = parseInt(delay)
 
+	let trail = d.trail?.map((t) => ({
+		latitude: t.lat ?? null,
+		longitude: t.lng ?? null,
+		altitude: t.alt ?? null,
+		bearing: t.hd ?? null, // in degrees
+		speed: t.spd ?? null,
+		timestamp: t.ts ?? null,
+	}));
```

Comment:
```suggestion
	})) ?? null;
```

Reply:
My intention was that a lack of trail data return [], as the covered case in line 98:

rail: trail || []

This was so the results could be handled with a trail.forEach() without first checking for null.

Do you consider null to be a better choice?